lib/components/providers/theme/themeContext.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { type Dispatch, type SetStateAction } from "react";
+
+export const initialThemeState = {
+    theme: `dark`,
```

Comment:
```suggestion
    theme: 'dark',
```
Not sure why are we using backticks instead of regular `' '` 🤔

Reply:
removed it